### DISCUSSION TITLE
Monthly release cycle

### DISCUSSION THREAD

**Discussion by rsj-ioki** [helix-editor/helix#7341] [General] [answered]
As a non-contributing user I think release cycle should be shortened to monthly. For example first week of the month sees a release of everything done in last month. My reasons:
- The editor even though quite complete is still under heavy development so quite a few changes get pushed in 2-3 months
- Its easier to track down changes in small release and less surprises
- Non-contributing users can provide feedback faster and more frequently
- Non-contributing users can get access to changes faster without having to maintain rust toolchain. For example I want [this](https://github.com/helix-editor/helix/issues/7126) but I would have to wait for quite sometime unless I build from source.

So if releasing is not a complex complex task and does not involve too much effort then in my opinion shorter release cycles are better. Of course this does not has to be permanent and once things are more stable then release cycles can be extended again.

**Comment by the-mikedavis** [comment, marked as answer]
See https://github.com/helix-editor/helix/discussions/6362#discussioncomment-5365452, we don't have any plans to change the release cycle.
